tests/acceptance/gcp-auth.bat

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bats
+
+#load _helpers
```

Comment:
do we need this line?

Reply:
I kept this line as well as the commented `SKIP_TEARDOWN` one to let any future devs who pick up these tests know that these are toggles that are available to them to switch, but I'm not too attached to either approach and am happy to remove if it seems more confusing than helpful